sdk/core/core-client-rest/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Upgrade dependency `@azure/abort-controller` to `^2.0.0`.
```

Comment:
All looks great to me 👍 Out of curiosity, why do we need to separate the dependency upgrade to 1.2.0 version instead of bundling it with the 1.1.7 release this morning?

Reply:
abort-controller v2 wasn't released yet.